configs/node-collector/journal/filter-tagging.conf

```diff
@@ -0,0 +1,6 @@
+<filter **>
+  @type record_transformer
+  <record>
+    tags "ADD-SPACE-DELIMITED-TAGS-HERE"
```

Comment:
You generally may want different tags there.
They represent different flows. this one is for stuff labelled `journal`, the other one is generally for the stuff emitted by fluentd itself.
I don't know how to not duplicate this code in a nice way - same for the output/forward also a duplication there.